src/distribution/value.coffee

```diff
@@ -125,10 +130,18 @@ module.exports = do ->
     switch choice_index
 
       when FIRST_CHOICE
-        return domain_create_value domain_get_value_of_first_contained_value_in_list fdvar.dom, list
+        v = domain_get_value_of_first_contained_value_in_list fdvar.dom, list
```

Comment:
The original function name was much shorter, and it was super ambiguous what it actually did. This eventually led to this name (there are, or at least were, similar functions). They are completely minified away, at least :)